app.py

```diff
@@ -0,0 +1,61 @@
+import os
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+from models import User
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
```

Comment:
Did we want to keep this Hello World test path?

Reply:
Maybe we'll keep it there to make sure things are working and then remove it later?